### ISSUE TITLE
seems no update job method

### ISSUE THREAD

**Issue by chenwenhao03** [bndr/gojenkins#22]
hi! it seems no update job method, if has pls show me,if not pls let me know.Might I can post a pr to u

**Comment by bndr** [comment]
You are correct there is no update job method, there is only rename, copy or create job.

Feel free to submit a pull request

**Comment by xkkker** [comment]
@bndr hi, I have created a PR to solve this issue.
If it's right please accept the PR.Thanks!
pls see [https://github.com/bndr/gojenkins/pull/198](url)

**Comment by StantonMatt** [comment]
I checked current `master`, and this seems implemented now: `Job.UpdateConfig(ctx, config)` exists and posts the supplied XML to the job `config.xml` endpoint through `Requester.PostXML`.

There is also current unit coverage for the success and error paths in `job_test.go` (`TestJob_UpdateConfig_Success` and `TestJob_UpdateConfig_Error`). Unless this issue meant a different update shape than replacing the job XML config, it can probably be closed.
